samples/sampler/stories/qa/Scroller.js

```diff
@@ -633,3 +634,40 @@ select('focusableScrollbar', WithCustomizedStyle, prop.focusableScrollbarOption,
 select('scrollMode', WithCustomizedStyle, prop.scrollModeOption, Config);
 
 WithCustomizedStyle.storyName = 'With Customized Style';
+
+export const WithLongContents = (args) => {
+	const content = `
+	The goal of Enact is to provide the building blocks for creating robust and maintainable applications. To that end, we’ve pulled together the best solutions for internationalization (i18n), accessibility (a11y), focus management, linting, testing and building. Then, we created a set of reusable components and behaviors on top of that. We combined these pieces and ensured that they work together seamlessly, allowing developers to focus on implementation.
+	Easy to Use
+	Enact builds atop the excellent React library, and provides a full framework to the developer. The recent boom of web technologies and related tools has led to a plethora of options available. In fact, getting started might be the most difficult part of building a modern web application.
+	Performant
+	Beyond initial setup, Enact continues to provide benefits. It was built with performance in mind, and conscious decisions were made to ensure that applications remain performant as they grow in size and complexity. This ranges from the way components are rendered to how data flows through application.
+	Customizable
+	Enact has a full set of customizable widgets that can be tuned and tweaked to the particular style of each project. Using our experience in building full UI libraries for a broad swath of devices ranging from TVs to watches, we have created a widget library whose components can easily be composed to create complex views and applications.
+	Adaptable
+	Enact was designed to produce native quality applications for a wide variety embedded web platforms. Read about Enact’s use cases and how it helps solve problems for Automotive, Robotics, TV and more.
+	`;
+	let longContents = '';
+	for (let i = 0; i < 50; i++) {
+		longContents += content;
+	}
```

Comment:
```suggestion
	const longContents = content.repeat(50);
```
Let's make it shorter.